UPGRADE-6.0.md

```diff
@@ -23,6 +23,7 @@ Console
 -------
 
  * `Command::setHidden()` has a default value (`true`) for `$hidden` parameter
+ * Remove `Helper::strlen()`, use `Helper::width()` instead.
```

Comment:
The `Deprecate` equivalent in `UPGRADE-5.3.md` should also be added, right?

Reply:
Thank you